baseframe/templates/baseframe/mui/forms.html.jinja2

```diff
@@ -101,13 +101,26 @@
   {%- endif %}
 {%- endmacro %}
 
-{% macro rendersubmit(buttons=[], css_class="", tabindex='', cancel_url='', style='horiz', csrf_error=false) -%}
+{% macro rendersubmit(form, ref_id, buttons=[], css_class="", tabindex='', cancel_url='', style='horiz', csrf_error=false) -%}
```

Comment:
Changing the parameters will break a lot of code. Add this to the end as optional values. https://github.com/search?q=org%3Ahasgeek+rendersubmit&type=Code